pkg/inject/inject.go

```diff
@@ -465,6 +477,9 @@ func (conf *ResourceConfig) injectPodSpec(patch *Patch) {
 		LivenessProbe:  conf.proxyLivenessProbe(),
 	}
 
+	if conf.debugSidecar != nil {
+		patch.addContainer(conf.debugSidecar)
```

Comment:
nit: this code block is kind of in the middle of the logic that builds and adds `sidecar` to the patch. mind moving it above or below?